index.html

```diff
@@ -3,14 +3,76 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+ 
+
 
 </head>
 
-<body>
+
+
+
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+<body>
+    <div class="container">
+    <nav>
+        <a class="menu" href="#"> Services</a> 
+        <a class ="menu" href="#">  Product</a>
+        <a class ="menu" href="#">  Vision</a>
+        <a  class ="menu" href="#"> Features</a>
+        <a class ="menu" href="#">  About</a>
+        <a class ="menu" href="#"> Contact</a>
+      </nav>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          <h1> Innovation </br>
+          On </br>
```

Comment:
Try to avoid hard coding the `<br>` -- instead add another div and make it a flex box where you can `justify-content: center;` or `text-align: center;`